lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -221,9 +303,47 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             , expectField (#fee . #getQuantity) (`shouldBe` expectedFee)
             , expectField (#coinSelection . #withdrawals) (`shouldSatisfy` (not . null))
             ]
-        -- TODO: sign and submit tx,
-        --       check that reward account is 0 on external rewardWallet,
-        --       make sure wa wallet balance is increased by withdrawalAmt - fee
+
+        apiTx <- case (rTx :: (HTTP.Status, Either RequestException (ApiConstructTransaction n))) of
+            (_, Left e) -> throwIO e
+            (_, Right tx) -> pure tx
+
+        let sealedTx = transaction apiTx
+
+        -- Sign tx
+        let toSign = Json [json|
+                               { "transaction": #{sealedTx}
+                               , "passphrase": #{fixturePassphrase}
+                               }|]
+        let signEndpoint = Link.signTransaction @'Shelley wa
+        signedTx <- getFromResponse #transaction <$>
+            request @ApiSignedTransaction ctx signEndpoint Default toSign
+
+        -- Submit tx
+        submitTx ctx signedTx [ expectResponseCode HTTP.status202 ]
```

Comment:
This one fails with:

```
The submitted transaction was rejected by the local node. Here's an error message that may help with debugging: TxValidationErrorInMode (ShelleyTxValidationError ShelleyBasedEraAlonzo (ApplyTxError [DelegsFailure (WithdrawalsNotInRewardsDELEGS (fromList [(RewardAcnt {getRwdNetwork = Mainnet, getRwdCred = KeyHashObj (KeyHash "89fc61d21ddfcbd4d43652bf05c40c346fa794871423b65052d7614c")},Coin 1000000000000)]))])) AlonzoEraInCardanoMode
```

I think that means it's a bug in the implementation?

Reply:
I think that withdrawal from external wallet may not be supported on construct tx ep (https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/constructTransaction) it used to be a thing for old tx ep (https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postTransaction -> redemption) as a feature to allow users to withdraw funds from ITN wallets.

But if that's the case the construct ep should not allow mnemonics in the first place, so yeah it is a bug in implementation.